NuGet.config

```diff
@@ -13,7 +13,6 @@
     <add key="darc-pub-dotnet-emsdk-2674f58" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-emsdk-2674f580/nuget/v3/index.json" />
     <!--  End: Package sources from dotnet-emsdk -->
     <!--  Begin: Package sources from dotnet-maui -->
-    <add key="darc-pub-dotnet-maui-c2482dd" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-maui-c2482dda/nuget/v3/index.json" />
```

Comment:
Assuming all of these config/version changes were done by darc so we should just accept them?

Reply:
yeah, I don't mess with changes darc made.